deployments/arbitrum/usdc.n/configuration.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "Compound USDC(native)",
+  "symbol": "cUSDCNv3",
+  "baseToken": "USDC",
+  "baseTokenAddress": "0xaf88d065e77c8cC2239327C5EDb3A432268e5831",
+  "baseTokenPriceFeed": "0x50834F3163758fcC1Df9973b6e91f0F0F0434aD3",
+  "borrowMin": "100e6",
+  "pauseGuardian": "0x78E6317DD6D43DdbDa00Dce32C2CbaFc99361a9d",
+  "storeFrontPriceFactor": 0.8,
+  "targetReserves": "5000000e6",
+  "rates": {
+    "supplyKink": 0.8,
+    "supplySlopeLow": 0.0325,
+    "supplySlopeHigh": 0.4,
+    "supplyBase": 0,
+    "borrowKink": 0.8,
+    "borrowSlopeLow": 0.035,
+    "borrowSlopeHigh": 0.25,
+    "borrowBase": 0.015
+  },
+  "tracking": {
+    "indexScale": "1e15",
+    "baseSupplySpeed": "0.000402083333333e15",
+    "baseBorrowSpeed": "0e15",
+    "baseMinForRewards": "10000e6"
+  },
+  "assets": {
+    "ARB": {
+      "address": "0x912ce59144191c1204e64559fe8253a0e49e6548",
+      "priceFeed": "0xb2A824043730FE05F3DA2efaFa1CBbe83fa548D6",
+      "decimals": "18",
+      "borrowCF": 0.55,
+      "liquidateCF": 0.60,
+      "liquidationFactor": 0.93,
+      "supplyCap": "0e18"
+    },
+    "GMX": {
```

Comment:
I wonder if it's worth listing `GMX` on this deployment as well. There's only $72K of GMX supplied in the first market.

Reply:
Ok let me remove it.  (This is mainly borrowed from original USDC, was thinking the market would be identical between two type of USDC

Reply:
I meant that more as an open question, whether it's worth adding it again

Reply:
IMO this is supposed to be a replacement of the USDC.e market so we should keep the configuration mostly the same.